projects/RabbitMQ.Client/client/impl/TcpClientAdapter.cs

```diff
@@ -27,7 +27,11 @@ public virtual async Task ConnectAsync(string host, int port)
                 throw new ArgumentException("No ip address could be resolved for " + host);
             }
 
+#if NETSTANDARD
```

Comment:
I could also reverse this and make it be `#if NET461` instead, which would mean any newer version of the framework would automatically use the better API.

This only really matters if the `#ifdef ` is overlooked when the TFM is updated.

Reply:
Choose whatever will be more noticeable in a year when we re-visit this library for `7.0.0`. I'm hoping it won't be that long but you never know...

Reply:
I went ahead and pushed up a change.